js/main.js

```diff
@@ -21,23 +22,91 @@ function getData() {
 }
 getData();
 
-var likedThought = document.querySelector('#form-input');
 var text;
 var authorName;
 
-function handleClick(event) {
+function handleLike(event) {
   event.preventDefault();
 
   var savedQuote = {
     quote: text,
-    author: authorName
+    author: authorName,
+    entryId: data.nextEntryId
   };
 
   data.nextEntryId++;
-  likedThought.reset();
-  data.entries.unshift(likedThought);
-  ul.prepend(savedQuote);
+  // likedThought.reset();
+  data.entries.unshift(savedQuote);
+  renderLiked(savedQuote);
+  viewSwap(savedQuote);
+}
+
+if ($button != null) {
+  $button.addEventListener('click', handleLike);
+}
+
+function renderLiked(entry) {
+  var card = document.createElement('li');
+  // card.className('card');
+  card.setAttribute('class', 'card');
+
+  var text = document.createElement('li');
+  text.textContent = entry.text;
+
+  var person = document.createElement('li');
+  person.textContent = entry.person;
+
+  ul.appendChild(card);
+  ul.appendChild(text);
+  ul.appendChild(person);
 
 }
 
-likedThought.addEventListener('click', handleClick);
+var navLiked = document.querySelector('.nav-liked-thoughts');
+navLiked.addEventListener('click', viewSwap);
+
+function viewSwap(savedQuote) {
+  var parent = document.querySelector('.content');
+  parent.innerHTML = '';
```

Comment:
This is bad news. If you are trying to delete all child elements, I suggest using the `replaceChildren` method with no arguments.

https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren